content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+![CMake and CTest workflow](/images/user-guide/testing/getting-started/cmake_ctest_workflow.png)
+
+## CTest
+
+[CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html) is similar to other test frameworks like GoogleTest, PyTest, or JUnit; though it uses a higher-level generic interface. CTest registers each test as a string of commandline arguments, and reports on whether invoking those arguments succeeded (returned zero) without hanging. CTest can also coordinate running tests in parallel processes. Unlike other test runners, CTest is programming language agnostic by interfacing with the OS shell. However this means a notable feature is absent from CTest: other test frameworks typically provide a lower-level interface which directly invokes functions inside a specific programming language. To provide this feature, CTest is used to invoke lower-level test runners.
+
+To simplify using function-level test reporting, O3DE provides wrapper code to register tests for frameworks such as GoogleTest and PyTest into CTest. These wrappers enable writing tests which run on any operating system. XML files are generated to track the low-level results not reported in CTest, as well as artifacts such as log output and crash dumps.
+
+In short, all tests in O3DE use CTest as a high level test coordinator. When executed, CTest calls other lower-level test tools and then reports on their success.
+
+![CTest calls other test runners](/images/user-guide/testing/getting-started/ctest_to_runners.png)
+
+### Starting CTest
+
+CTest expects its working directory to be a CMake build directory, so be sure to first navigate to this directory:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration>
+```
+
+You may not want to run the commands above without also filtering the set of tests to run. Without a filter the command will run every test, and likely result in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing `CTRL + C` . The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following ctest command filters to only tests in these two suites:
+
+```shell
+cd <local_path_to>/o3de/build/<build_folder>
+ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
+```
+
+The build configuration should be one you have already built locally. Running the Smoke and Main suites for the `profile` build configuration on a Linux machine should look like:
+
+```shell
+cd user/github/o3de/build/linux
+ctest -C profile -L "(SUITE_smoke|SUITE_main)"
+```
+
+After running CTest, results save to `.../<build_folder>/Testing/`.  If you prefer to see a summary of failures in your terminal, add the flag `--output-on-failure`.
```

Comment:
I have some suggestions for organizing this section. 
- Split up the instructions for Windows and Linux. This makes it easier for users to find the instructions for their machine. Also, by using **On Windows** and **On Linux** headers, you can cross link to them if you need to in future docs. 
- Write the instructions in steps (1, 2, etc.)
   - Created Step 2: "Use CTest to run your tests." Then, put the two instructions for running CTest with filters and without into sub-bullets. 
- Moved the statement about `<build_configuration>` after you first introduce it. 

Visually, the suggestion would look like this: 
![image](https://user-images.githubusercontent.com/75444793/153935365-76315e77-3bac-4d92-9a61-4fd495b4bf92.png)


```suggestion

#### On Windows

1. In a terminal, navigate to the CMake build directory, which is CTest's expected working directory.

    \```shell
    cd <local_path_to>/o3de/build/<build_folder>
    \```

2. Use CTest to run your tests. 
   
   - You can run the following `ctest` command. This runs every test and likely results in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing **CTRL**+**C**.

        \```shell
        ctest -C <build_configuration>
        \```

        - `<build_configuration>` should be one that you've already built locally.

    - Alternatively, you can filter the set of tests to run. The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following `ctest` command filters to only test in these two suites:

        \```shell
        cd <local_path_to>/o3de/build/<build_folder>
        ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
        \```

#### On Linux

For a Linux machine, use the following commands to navigate to the `linux` CMake build directory and run the Smoke and Main suites for the `profile` build configuration.

\```shell
cd user/github/o3de/build/linux
ctest -C profile -L "(SUITE_smoke|SUITE_main)"
\```

After running CTest, the results save to `.../<build_folder>/Testing/`.  If you prefer to see a summary of failures in your terminal, run the `ctest` command with the the flag `--output-on-failure`.
```

Reply:
These are some great suggestions.

Reply:
This is an interesting suggestion, though there are far more similarities on Windows and Linux than differences. Trying to find a narrower split to describe only the differences.

Reply:
Another way is for each step, specify **Windows** or **Linux** if they are different, or don't specify either if they're the same. 
For example: 

1. In a terminal, navigate to the CMake build directory, which is CTest's expected working directory.
   - **Windows**:
      ```shell
      cd <local_path_to>/o3de/build/<build_folder>
      ```
   - **Linux**:
      ```shell
      cd user/github/o3de/build/linux
      ```
2. Use CTest to run your tests. The commands are similar for both Windows and Linux.
   - You can run the following `ctest` command. This runs every test and likely results in multiple hours of test execution! If you ever want to stop CTest, send an interrupt signal by selecting the terminal and pressing **CTRL**+**C**.
        ```shell
        ctest -C <build_configuration>
        ```
        - `<build_configuration>` should be one that you've already built locally.
    - Alternatively, you can filter the set of tests to run. The most commonly used suites are Smoke and Main, which have requirements to execute fast. The following `ctest` command filters to only test in these two suites:
        ```shell
        ctest -C <build_configuration> -L "(SUITE_smoke|SUITE_main)"
        ```